### PR TITLE
Template updated and system time updated

### PR DESCRIPTION
**Pull request by Radheshhathwar**

### CODE REVIEW

File: src/main/java/org/upsmf/grievance/service/impl/TicketServiceImpl.java [hunk 1]
```diff
@@ -90,6 +92,8 @@ public Ticket saveWithAttachment(Ticket ticket, List<String> attachments) {
         org.upsmf.grievance.model.es.Ticket esticket = convertToESTicketObj(ticket);
         // save ticket in ES
         esTicketRepository.save(esticket);
+        System.out.println(esticket);
+        System.out.println(psqlTicket);
```
Comment (shishirsuman092) [inline src/main/java/org/upsmf/grievance/service/impl/TicketServiceImpl.java:96]: remove

File: src/main/java/org/upsmf/grievance/service/impl/TicketServiceImpl.java [hunk 2]
```diff
@@ -127,8 +131,9 @@ public Ticket save(TicketRequest ticketRequest) throws Exception {
         // create ticket
         ticket = saveWithAttachment(ticket, ticketRequest.getAttachmentUrls());
         // send mail
-        EmailDetails emailDetails = EmailDetails.builder().recipient(ticket.getEmail()).subject("New Complaint Registration").build();
-        emailService.sendCreateTicketMail(emailDetails, ticket);
+        //EmailDetails emailDetails = EmailDetails.builder().recipient(ticket.getEmail()).subject("New Complaint Registration").build();
+        //emailService.sendCreateTicketMail(emailDetails, ticket);
+        System.out.println(ticket);
```
Comment (shishirsuman092) [inline src/main/java/org/upsmf/grievance/service/impl/TicketServiceImpl.java:136]: why is this commented

File: src/main/java/org/upsmf/grievance/controller/TicketController.java [hunk 3]
```diff
@@ -24,7 +24,7 @@ public class TicketController {
     private AttachmentService attachmentService;
 
     @PostMapping("/save")
-    public ResponseEntity<Response> save(@RequestBody TicketRequest ticketRequest) {
+    public ResponseEntity<Ticket> save(@RequestBody TicketRequest ticketRequest) {
```
Comment (shishirsuman092) [inline src/main/java/org/upsmf/grievance/controller/TicketController.java:27]: remove

File: src/main/java/org/upsmf/grievance/controller/TicketController.java [hunk 4]
```diff
@@ -33,7 +33,7 @@ public ResponseEntity<Response> save(@RequestBody TicketRequest ticketRequest) {
             throw new RuntimeException(e.getMessage());
         }
         Response response = new Response(HttpStatus.OK.value(), responseTicket);
-        return new ResponseEntity<Response>(response, HttpStatus.OK);
+        return new ResponseEntity<>(responseTicket, HttpStatus.OK);
```
Comment (shishirsuman092) [inline src/main/java/org/upsmf/grievance/controller/TicketController.java:36]: remove

**Review by shishirsuman092 — CHANGES_REQUESTED** [review]

File: src/main/java/org/upsmf/grievance/controller/TicketController.java (hunk 3 above)
Comment (Radheshhathwar) [reply src/main/java/org/upsmf/grievance/controller/TicketController.java:27]: Changed

File: src/main/java/org/upsmf/grievance/service/impl/TicketServiceImpl.java [hunk 5]
```diff
@@ -129,6 +131,7 @@ public Ticket save(TicketRequest ticketRequest) throws Exception {
         // send mail
         EmailDetails emailDetails = EmailDetails.builder().recipient(ticket.getEmail()).subject("New Complaint Registration").build();
         emailService.sendCreateTicketMail(emailDetails, ticket);
+        System.out.println(ticket);
```
Comment (shishirsuman092) [inline src/main/java/org/upsmf/grievance/service/impl/TicketServiceImpl.java:134]: remove

**Review by shishirsuman092 — CHANGES_REQUESTED** [review]

**Pull request merged**